src/util.h

```diff
@@ -113,13 +116,50 @@ bool Truncate(const std::string& path, size_t size, std::string* err);
 #ifdef _MSC_VER
 #define snprintf _snprintf
 #define fileno _fileno
+#define strtoull _strtoui64
+
+#ifdef UNICODE
+#define unlink _wunlink
+#define fopen(path, mode) _wfopen(path, TEXT(mode))
+#define chdir _wchdir
+#define getcwd _wgetcwd
+#define mkdir _wmkdir
+
+#define t_snprintf _snwprintf
+#define t_splitpath _wsplitpath
+#define t_stricmp _wcsicmp
+#define t_strlen wcslen
+
+#define PATH_MAX 2048
+#else
```

Comment:
the multiple `#else` and `#endif` directives are a little hard to follow. Can you use comment to clarify them a little, e.g.:

```
#ifdef _MSC_VER
....
#ifdef UNICODE
...
#else  // !UNICODE
...
#endif // !UNICODE
#else // !_MSC_VER
...
#endif // !_MSC_VER
```